docs/_docs/thin-clients/java-thin-client.adoc

```diff
@@ -79,6 +79,10 @@ Note that the code above provides a failover mechanism in case of server node fa
 
 include::includes/partition-awareness.adoc[]
 
+Partition awareness functionality helps to avoid an additional network hop in the following scenarios:
+1. Single-key operations API, like #put, #get, etc. However, the functionality has no effect on those operations within explicit transactions (initiated via ClientTransaction.txStart() described in <<Executed transactions>> section).
```

Comment:
I set up docs locally and link to "Executing Scan Queries" works, but links on transactions and IndexQueries doesn't. Could you, please, check it?

Reply:
I've corrected the links to Transactions and IndexQueries
Thank you